rskj-core/src/test/java/org/ethereum/core/NestedCallsStackDepthTest.java

```diff
@@ -0,0 +1,111 @@
+package org.ethereum.core;
+
+import co.rsk.config.TestSystemProperties;
+import co.rsk.core.ReversibleTransactionExecutor;
+import co.rsk.core.RskAddress;
+import co.rsk.core.TransactionExecutorFactory;
+import co.rsk.rpc.ExecutionBlockRetriever;
+import co.rsk.rpc.modules.eth.EthModule;
+import co.rsk.test.World;
+import co.rsk.test.dsl.DslParser;
+import co.rsk.test.dsl.DslProcessorException;
+import co.rsk.test.dsl.WorldDslProcessor;
+import org.bouncycastle.util.encoders.Hex;
+import org.ethereum.config.Constants;
+import org.ethereum.rpc.CallArguments;
+import org.ethereum.rpc.parameters.BlockIdentifierParam;
+import org.ethereum.util.TransactionFactoryHelper;
+import org.ethereum.vm.PrecompiledContracts;
+import org.ethereum.vm.program.invoke.ProgramInvokeFactoryImpl;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.io.FileNotFoundException;
+
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+
+public class NestedCallsStackDepthTest {
+
+    private static final CallTransaction.Function CALL_CONTRACTB_FUNCTION = CallTransaction.Function.fromSignature("callContractB");
+
+    private World world;
+    private WorldDslProcessor processor;
+    private EthModule ethModule;
+
+    /** ------------------------ **
+     *  SETUP
+     ** ------------------------ **/
+    @BeforeEach
+    void setup() {
+        world = new World();
+        processor = new WorldDslProcessor(world);
+        ethModule = buildEthModule(world);
```

Comment:
have you considered this one - `EthModuleTestUtils.buildBasicEthModule(world)`?

Reply:
Done, I noticed that I could just get the `ethModule` directly from the method you suggested and remove the `buildEthModule` method.